keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlVerificationRepository.kt

```diff
@@ -196,7 +196,7 @@ class SqlVerificationRepository(
         }
         .run {
           if (metadata.isNotEmpty()) {
-            set(VERIFICATION_STATE.METADATA, jsonSet(VERIFICATION_STATE.METADATA, metadata))
+            set(VERIFICATION_STATE.METADATA, jsonMerge(VERIFICATION_STATE.METADATA, metadata))
```

Comment:
These MySQL functions differ in that `json_set` would overwrite an array property, but `json_merge` will append new elements to an existing array